### DISCUSSION TITLE
Would it be possible for drag drop demo to drop jquery-ui ?

### DISCUSSION THREAD

**Discussion by giannik** [selectize/selectize.js#1733] [Ideas]
Im currently evaluating drag drop demo for re-ordering selected items as shown here :
https://selectize.dev/plugins/2020/01/01/drag-drop/

Would it be possible to drop jquery-ui and use a more lightwight library like sortable js  for the reordering of selected items ?
Jquery-ui is quite a big file and Im looking to make it more lightweight in terms of file size.
Thanks

**Comment by risadams** [comment]
Certainly open to the idea, and would welcome a pull request.
The big issues we would need to consider are not breaking existing users.
Ideally, this would involve a plugin for each, and the implementing users would be able to select which (if any) plugin to use during config

**Comment by coudrak** [comment]
Tom Select is a fork of Selectize without jQuery.
You can use it: https://tom-select.js.org/plugins/drag-drop/

**Reply by fabienwnklr** [reply]
Hm tom-select don't use jquery but this plugin use it :)

**Reply by coudrak** [reply]
This plugin exists without jquery for tom-select: https://tom-select.js.org/plugins/drag-drop/

**Reply by fabienwnklr** [reply]
i would insist, plugin required jQuery and jQueryUI
 
![image](https://user-images.githubusercontent.com/49644785/190598842-907a6d7e-06f1-453f-bcb5-9bc0b136da7f.png)
